### ISSUE TITLE
Porting to Material design little - mdl

### ISSUE THREAD

**Issue by srlopez** [bradwestfall/CSS-Tricks-React-Series#3]
Nice article, nice pictures, nice css ... nice all.

but i'd like go a little bit forward.
Just to ask if it will be a next article on porting your stuff to mdl, or any other material design css frameworks.
I am really frustrated on my own work to merge react-router with this framework.

I have change the index like this [index](https://getmdl.io/started/index.html)

On main-layout i translated some of the samples in [this](page), but i don't know how to link the menu with the Link component of react-router, and show the result in the 'container'

Thanks in advance

**Comment by bradwestfall** [comment]
Thanks @srlopez, but I won't be doing the series beyond what we have. It was just meant to be a 3-part series. Also, I'm not fully sure what you mean by the router not working with trying to port the code to some other design stuff like Material Design. I wouldn't think the router has much to do with that

**Comment by mchavezi** [comment]
" i don't know how to link the menu with the Link component of react-router, and show the result in the 'container'" 

I think I understand what @srlopez is saying, I think I am trying to do the same thing.

I have a teamMemberList and teamMemberProfile, based of the user component and would like to have the teamMemberList view inside the teamMemberProfile, like https://github.com/reactjs/react-router/tree/master/examples/master-detail. 

Been trying for days to get this to work with what I have already started using this tutorial, so I don't want to give up. 

I posted my question on stackoverflow. Any help would be greatly apprenticed. 
http://stackoverflow.com/questions/39234374/updating-component-with-react-router-link-redux-and-componentwillreceiveprops-c
